client/app/states/services/list/list.state.js

```diff
@@ -82,6 +84,33 @@
       sortConfig: serviceSortConfig,
     };
 
+    vm.actionButtons = [
+      {
+        name: __('Start'),
```

Comment:
Both `N_('Start')` and `__('Start')` are valid in javascript sources.
Notice that the blueprints js uses the `__('something')` format here : 
https://github.com/ManageIQ/manageiq-ui-service/blob/master/client/app/components/blueprints/blueprints-list.directive.js#L65